token/program-2022/src/extension/mod.rs

```diff
@@ -77,7 +81,7 @@ fn get_extension_indices<V: Extension>(
     let v_account_type = V::TYPE.get_account_type();
     while start_index < tlv_data.len() {
         let tlv_indices = get_tlv_indices(start_index);
-        if tlv_data.len() <= tlv_indices.value_start {
+        if tlv_data.len() < tlv_indices.value_start {
```

Comment:
🙋🏼  `<=` to `<` was needed to support extensions that hold no data, like the `FixedOwner` extension